lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -23,12 +23,42 @@
     border-top: 3px solid $blue;
     padding: $baseline 0;
 
-    .copy {
-      @extend %t-copy-sub1;
-    }
-
-    .btn-find-courses {
-      @extend %btn-pl-elevated-alt;
+    .course-advertise {
+      padding: $baseline;
+      background-color: $body-bg;
+      @include box-sizing(border-box);
+      border: 1px solid $border-color-l3;
+      @include clearfix();
```

Comment:
Note that all @include and @extend's should be called at the top of the selector